roles/lnls-ans-role-pydm/molecule/default/molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
+dependency:
+  name: galaxy
+lint:
+  name: yamllint
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+    options:
+      x: ["ANSIBLE0016"]
+  inventory:
+    group_vars:
+      default_group:
+scenario:
+  name: default
+verifier:
+  name: testinfra
+  options:
+    verbose: true
+  lint:
+    name: flake8
+driver:
+  name: docker
+platforms:
+  - name: lnls-ans-role-sirius-apps-instance
```

Comment:
@fernandohds564 , should `lnls-ans-role-sirius-apps-instance` be changed here?

Reply:
I don't know @xresende. Should I, @lerwys ?

Reply:
Yes, let's change it. Any name would work, but this name is printed in the test logs (travis), so it's better to use a more descriptive name

Reply:
done!